tox.ini

```diff
@@ -3,16 +3,16 @@ max-line-length = 100
 
 [tox]
 skipsdist = True
-envlist = {server}
+envlist = {mlflow-server}-{unit,lint,integration},selenium
```

Comment:
Interestingly, the `mlflow` here doesn't actually matter.  You can still run the commands without it (try `tox -e server-lint`).

Also, I think `fmt` is missing from the second list.  We can still use `tox -e server-fmt`, but that env wont show up in `tox -l` output unless we add it to this line.